client/nginx.conf

```diff
@@ -0,0 +1,37 @@
+# TODO move to separate sidecar container
+
+events {
+  worker_connections 768;
+}
+
+http {
+  # Nginx will handle gzip compression of responses from the app server
+  gzip on;
+  gzip_proxied any;
+  gzip_types text/plain application/json;
+  gzip_min_length 1000;
+
+  server {
+    listen 80;
+    root /usr/share/nginx/html;
+    include /etc/nginx/mime.types;
+
+    location /api {
+      # Reject requests with unsupported HTTP method
+      if ($request_method !~ ^(GET|POST|HEAD|OPTIONS|PUT|DELETE)$) {
```

Comment:
I think `PATCH` should also be supported with our current API, not sure if that will still be kept with data model v2 updates @Naraujo13 did you also consider PUT vs PATCH for all the data model updates in the API?

Reply:
Good call, I just copied/pasted this from an AWS sample code. I can add `PATCH` here

Reply:
yeah I guess `PATCH` is a lesser used HTTP method

Reply:
> I think `PATCH` should also be supported with our current API, not sure if that will still be kept with data model v2 updates @Naraujo13 did you also consider PUT vs PATCH for all the data model updates in the API?

Yeah, the API is using `PATCH` at the API updates. It was already using `PATCH` before and I kept that while updating the data models.